### PR TITLE
[IOCOM-1362] Added temporary model

### PR DESCRIPTION
**Pull request by michaeldisaro**
#### List of Changes
Added a temporary non versioned model with some defaults to get rc configurations as if they are non versioned.

#### Motivation and Context
We need to migrate to a non versioned model.

#### Types of changes

- [x] Chore (nothing changes by a user perspective)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

File: src/models/rc_configuration_non_versioned_temp.ts [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+    "hasPrecondition"
+  ),
+  isLollipopEnabled: t.boolean,
+  name: NonEmptyString,
+  userId: NonEmptyString
+});
+
+const RCConfigurationO = t.partial({
+  version: NonNegativeInteger
+});
```
Comment (Garma00) [inline src/models/rc_configuration_non_versioned_temp.ts:74]: Do we need this?

File: src/models/rc_configuration_non_versioned_temp.ts [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+
+export const RCConfigurationBase = t.intersection([
+  RCConfigurationR,
+  RCConfigurationO
```
Comment (Garma00) [inline src/models/rc_configuration_non_versioned_temp.ts:78]: Ditto

**Review by Garma00 — APPROVED** [review]
I was thinking if we do not need the versioned one any longer what about using this model in the unit test already defined for the versioned one (with fix if needed)?

File: src/models/rc_configuration_non_versioned_temp.ts (hunk 1 above)
Comment (michaeldisaro) [reply src/models/rc_configuration_non_versioned_temp.ts:74]: yep.

File: src/models/rc_configuration_non_versioned_temp.ts (hunk 2 above)
Comment (michaeldisaro) [reply src/models/rc_configuration_non_versioned_temp.ts:78]: yep

**Review by acuiuli — APPROVED** [review]

**Pull request merged**